docs/UserGuide.md

```diff
@@ -222,20 +222,20 @@ This command updates the bill of the guest at the specified `INDEX`.
 Depending on the sign ('+' or '-') of your input, the existing bill value be incremented 
 or decremented respectively by the value of your input.
 
-#### Format: `bill INDEX b/BILL`
+**Format**: `bill INDEX b/BILL`
 
-#### Constraints of parameters:
+**Constraints of parameters**:
 * The bill value **must be either a whole number or a whole number with up to 2 decimal places.**
 
-#### Examples:
+**Examples**:
 * `bill 1 b/+99.99` or `bill 1 b/99.99` Adds 99.99 to the bill of the first guest.
 * `bill 2 b/-10` Subtracts 10 from the bill of the second guest.
 
-#### Common mistakes:
+**Common mistakes**:
```

Comment:
Would it be nicer if "M" of mistakes is capitalised? If you feel it's fine, feel free to ignore this!

Reply:
Likewise for the same comment above!